app/assets/stylesheets/notifications.scss

```
@@ -67,11 +67,23 @@
               margin: 10px auto;
               width: 90%;
               color:#666666;
+              &.milestone-emojis{
+                font-size: xx-large;
+                padding-top: 10px;
+                padding-bottom: 0px;
+              }
             }
             img.badge-image{
               margin:10px 0px 20px;
               height: 150px;
             }
+            img.milestone-gif{
+                margin: 10px 0px 10px;
```

I think a border radius on the gif would be a nice touch.

```suggestion
                margin: 10px 0px 10px;
                border-radius: 8px;
```